resources/event-hackathon/notebook-authoring-checklist.md

```diff
@@ -0,0 +1,120 @@
+    - [ ] Title
+    - [ ] Axis labels and range
+    - [ ] Key or legend
+    - [ ] An explanation of the visualization's significance to the notebook (like the trend, an outlier in the data, what the author learned from it, etc.)
+
+- [ ] All visualizations have the following labels
+    - [ ] Title
+    - [ ] Labels on all axes
+    - [ ] Key or legend (if relevant)
+
+- [ ] All visualizations and their parts have enough color contrast to be legible. Remember that transparent colors have lower contrast than their opaque versions.
+- [ ] All visualizations convey information with more than color-coding. Color may also be mixed with text labels, patterns, or icons to fulfill this.
+- [ ] All visualizations have an additional way for notebook readers to access the information. Linking to the original data, including a table of the data in the same notebook, or sonifying the plot are all options.
+
+## Text
+
+- [ ] All link text is descriptive. It tells users where they will be taken if they open the link.
+- [ ] Use plain language wherever possible.
+- [ ] All acronyms are defined at least the first time they are used. 
+- [ ] Field-specific/specialized terms are used when needed, but not excessively.
+- [ ] Text is broken into paragraphs and/or cells where relevant.
+- [ ] Text is in complete sentences where relevant.
+
+## Code
+
+- [ ] Code sections are introduced/explained before they appear in the notebook. This could be a heading in a prior Markdown cell, a sentence preceding it, or a code comment in the code section.
+- [ ] Code has explanatory comments (if relevant). This is most important for long sections of code.
+- [ ] If the author has control over the syntax highlighting theme in the notebook, that theme has enough color contrast to be legible.
+- [ ] Code and code explanations focus on one task at a time. Unless comparison is the point of the notebook, only one way to complete the task is described at a time.
+
+## Other content
+
+This list is not exhaustive. If you are reviewing a notebook with content that you do not think fits any of these categories, keep in mind
+
+- Text is flexible. Whether it is in the document or linked out, text can be read visually, be read audibly, be magnified, or be translated to another language. Having a text alternative is a good back up plan.
+- Having enough color contrast is required on almost all visual content.
+
+### Videos
+
+- [ ] All videos have titles in the player or in the text before them.
+- [ ] All videos have captions/subtitles. This can include visual information descriptions if relevant.
+- [ ] All videos have transcripts. This can include visual information descriptions if relevant.
+- [ ] All video players have buttons with labels. This can be a persistent label or appear when hovered.
+- [ ] All video players have buttons with enough color contrast.
+- [ ] No videos have flashing images at more than three frames per second.
+
+### Sonifications
+
```

Comment:
- [ ] Sonifications include a key explaining the mapping of data to sound. A written description can be used to convey this information.

- [ ] Sonification outputs reference the method that generated the sonification. This can be done in a code cell or with a link to the file used to generate the sonification.

- [ ] Audio players include basic listening controls for starting, pausing, volume, and speed.